README.md

```diff
@@ -5,14 +5,22 @@
 [![Build Status][ci-img]][ci]
 [![Report Card][report-card-img]][report-card]
 
-## Status
+UberFx is a flexible, modularized framework for building robust and performant
+services. It removes boilerplate code out of your hands and lets you focus on
```

Comment:
It's awkward to say "removes…out of your hands." Some alternatives:
- It removes boilerplate from your to-do list and lets you focus on
- It removes boilerplate from your job, letting you focus on
- It takes care of the boilerplate code and lets you focus on

Feel free to mix and match.